gateway/webhook/webhookTransformer.go

```diff
@@ -109,18 +112,21 @@ func (v2 *v2Adapter) getAdapterVersion() string {
 
 // ------------------------------
 
-func newSourceTransformAdapter(version string) sourceTransformAdapter {
+func newSourceTransformAdapter(version string, conf *config.Config) sourceTransformAdapter {
 	// V0 Deprecation: this function returns v1 adapter by default, thereby deprecating v0
 	if version == transformer.V2 {
-		return &v2Adapter{}
+		return &v2Adapter{
+			baseTransformerURL: conf.GetString("DEST_TRANSFORM_URL", "http://localhost:9090"),
```

Comment:
[optional] the conf.GetString is duplicated, perhaps create a var before the ifs?